packages/pn-pa-webapp/src/pages/NotificationDetail.page.tsx

```diff
@@ -271,6 +279,20 @@ const NotificationDetail = () => {
     return () => void dispatch(resetState());
   }, [fetchSentNotification]);
 
+  /* function which loads relevant information about donwtimes */
+  const fetchDowntimeEvents = useCallback((fromDate: string, toDate: string | undefined) => {
+    const fetchParams: GetNotificationDowntimeEventsParams = {
+      startDate: fromDate,
+      endDate: toDate,
+    };
+    void dispatch(getDowntimeEvents(fetchParams));
+  }, []);
+
+  const fetchDowntimeLegalFactDocumentDetails = useCallback(
+    (legalFactId: string) => void dispatch(getDowntimeLegalFactDocumentDetails(legalFactId)),
+    [dispatch, getDowntimeLegalFactDocumentDetails]
```

Comment:
dispatch and getDowntimeLegalFactDocumentDetails never change. they can be removed as dependencies